### PR TITLE
Experimental multiselect

### PR DESCRIPTION
**Pull request by AndyBuchwald**

### CODE REVIEW

**Review by kinnarr — APPROVED** [review]
LGTM

**Review by bmario — APPROVED** [review]

**Comment by tilsche** [comment]
As far as I can tell `newEmptyMetric` is now unused, should be removed.

File: js/ui/new-metric-popup.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      }
+      veil.create(disablePopupFunc)
+      veil.attachPopup(popupEle)
+    }
+  },
+  methods: {
+    addMetrics: function (evt) {
+      if (this.value != null) {
+        this.value.forEach(function (item, index, array) {
+          Store.setMetric(item, new Metric(window.MetricQWebView.instances[0], item, undefined, markerSymbols[(index * 4) % markerSymbols.length], []))
```
Comment (tilsche) [inline js/ui/new-metric-popup.js:42]: I don't like the selection of markerSymbols at this point. I'd rather leave it `undefined` and have `Metric.constructor` pick one for us at "random" / based on the metric name, just like we do it with the color.

The same applies to the other point where we call `new Metric` - it's even worse there with no modulo wrap around.

**Comment by tilsche** [comment]
In usage, there is an issue that the drop-down hides the add button... that's quite fragile to use. Also having the drop down extend over the veil "window" is not great ui.

If possible, it would be great if we could just have the "drop down" always shown as a kind of multiline text box (with a fixed height and if needed scroll bar), and the "add" button place below.

**Comment by tilsche** [comment]
Another thing we talked about (but not in the issue)  was showing the metadata within the listing: description (possible cut to a certain length). Vue-Multiselect seems very flexible to allow this. I think it would be great if we could add this in the PR rather than later.

**Comment by tilsche** [comment]
Another minor nitpick. In the vue-multiselect examples it can be seen that selection are abbreviated if listing all the selected items at the top would become too long. This would be good to have for us as well.

Screenshot_2021-04-23_16-45-13

**Comment by tilsche** [comment]
Would it be possible to show the description only in the long vertical listing, but in the header with the selected only the name is  shown?

**Pull request merged**